[2021-05-09 18:00:17,669] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.move_files 2021-05-08T05:40:06.416028+00:00 [queued]>
[2021-05-09 18:00:17,701] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.move_files 2021-05-08T05:40:06.416028+00:00 [queued]>
[2021-05-09 18:00:17,701] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-09 18:00:17,701] {taskinstance.py:1069} INFO - Starting attempt 1 of 3
[2021-05-09 18:00:17,702] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-09 18:00:17,907] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): move_files> on 2021-05-08T05:40:06.416028+00:00
[2021-05-09 18:00:17,910] {standard_task_runner.py:52} INFO - Started process 6074 to run task
[2021-05-09 18:00:17,916] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_4', 'move_files', '2021-05-08T05:40:06.416028+00:00', '--job-id', '113', '--pool', 'default_pool', '--raw', '--subdir', '/home/navneetsajwan/airflow/dags/dag__4.py', '--cfg-path', '/tmp/tmp0yf37t1g', '--error-file', '/tmp/tmpowzno5zs']
[2021-05-09 18:00:17,917] {standard_task_runner.py:77} INFO - Job 113: Subtask move_files
[2021-05-09 18:00:18,050] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_4.move_files 2021-05-08T05:40:06.416028+00:00 [running]> on host navneetsajwan-Lenovo-G50-80
[2021-05-09 18:00:18,200] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=navneet@metaorigins.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag_4
AIRFLOW_CTX_TASK_ID=move_files
AIRFLOW_CTX_EXECUTION_DATE=2021-05-08T05:40:06.416028+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-05-08T05:40:06.416028+00:00
[2021-05-09 18:00:18,201] {python.py:118} INFO - Done. Returned value was: None
[2021-05-09 18:00:18,206] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=dag_4, task_id=move_files, execution_date=20210508T054006, start_date=20210509T123017, end_date=20210509T123018
[2021-05-09 18:00:18,505] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-05-09 18:00:18,527] {local_task_job.py:146} INFO - Task exited with return code 0
